### ISSUE TITLE
"Object reference not set to an instance of an object" dialog when pasting the following code into the editor (VS2013)

### ISSUE THREAD

**Issue by mousetraps** [Microsoft/nodejstools#549]
```js
'use strict'
let m = new Map()
m.set("hello", 42)
m.set(s, 34)
m.get(s) === 34
m.size === 2
for (let [ key, val ] of m.entries())
    console.log(key + " = " + val)
```

**Comment by paladique** [comment]
I copied and pasted that code line by line. Seems like it's something with that `for` loop. After that I get that dialog message every time I press 'Enter' to make a new line.

I started poking around a little more and started commenting out parts of that line and pressing 'Enter'. The dialog seems to come up after the `[` character.

**Comment by mjbvz** [comment]
I believe this was fixed for ES5 at some point (perhaps with #743), and I don't see it in ES6 IntelliSense with NTVS 1.2 Beta at all.

Please try upgrading to the beta if you are running into this problem. Thanks.
